.github/workflows/base-checks.yaml

```diff
@@ -8,13 +8,13 @@ on:
 
 jobs:
   basic-checks:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why did we need to pin the version here, but use ubuntu-latest below in sdk-scorecard.yaml?

Reply:
I think we need to use the same version for all our GH actions, My understanding is that it is usually a good practice to pin the versions. We can do it in a separate PR.

Reply:
I see there is currently a mix of ubuntu-latest and ubuntu-20.04 in various files. We can make these consistent in a separate PR.